base_view_inheritance_extension/models/ir_ui_view.py

```diff
@@ -189,6 +189,42 @@ def inheritance_handler_attributes_domain_add(self, source, specs):
             node.attrib[attribute_name] = new_value
         return source
 
+    @api.model
+    def inheritance_handler_attributes_attrs_domain_add(self, source, specs):
+        """Implement attrs_domain_add
+
+        <attribute name="$attribute" operation="attrs_domain_add"
+                   key="$attrs_key" join_operator="OR">
+            $domain_to_add_to_attrs_key
+        </attribute>
+        """
+        node = self.locate_node(source, specs)
+        for attribute_node in specs:
+            attribute_name = attribute_node.get("name")
+            key = attribute_node.get("key")
+            join_operator = attribute_node.get("join_operator") or "AND"
+            old_value = node.get(attribute_name) or ""
+            if old_value:
+                old_value = ast.literal_eval(
+                    self.var2str_domain_text(old_value.strip())
+                )
+                old_domain_attrs = old_value.get(key)
+                new_domain = ast.literal_eval(
+                    self.var2str_domain_text(attribute_node.text.strip())
```

Comment:
I would first normalize the domain just in case before appending new ones with the method normalize_domain of expression

Reply:
I think that odoo does a normalization when in the call to OR or AND https://github.com/odoo/odoo/blob/16.0/odoo/osv/expression.py#L264

Reply:
Yes, there seems to be a discrepancy here between the documentation in the doc-string and the implementation. @BT-etejeda since you found it maybe you can do a PR to Odoo to correct it.